pymc_marketing/clv/models/beta_geo.py

```diff
@@ -437,6 +437,71 @@ def expected_probability_alive(
             "chain", "draw", "customer_id", missing_dims="ignore"
         )
 
+    def expected_probability_no_purchase(
```

Comment:
Can we rename this to `expected_purchase_probability` for consistency with `ParetoNBDModel`? This method can be used for more than just the case of x=0.